js/adapt-stateful-session.js

```diff
@@ -127,6 +145,16 @@ define([
 			Adapt.offlineStorage.set("interaction", id, response, result, latency, responseType);
 		},
 
+		onResetSession: function () {
+			this.removeEventListeners();
+			this.setupEventListeners();
```

Comment:
When you load a new language, it creates a new blocks, course, components ... Collection from the new JSON data. The Collection can be different for each language. So I thought it would be better to remove and reattach the eventListener.